integration-libs/cdc/root/guards/cdc-user.guard.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { CanActivate, Router, UrlTree } from '@angular/router';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  SemanticPathService,
+} from '@spartacus/core';
+import { UserGuard } from '@spartacus/organization/administration/core';
+import { CdcB2BUserService } from '../../organization/administration/public_api';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CdcUserGuard extends UserGuard implements CanActivate {
+  constructor(
+    protected globalMessageService: GlobalMessageService,
+    protected cdcb2bUserService: CdcB2BUserService,
```

Comment:
Maybe let's rename this parameter to `cdcB2bUserService`. Just to keep naming convention consistency.

Reply:
done